inc/inventorynetworkequipmentlib.class.php

```diff
@@ -253,7 +257,7 @@ function internalPorts($a_ips, $networkequipments_id, $mac, $networkname_name) {
     * @param array $a_inventory
     * @param integer $items_id
     */
-   function importPorts($a_inventory, $items_id) {
+   function importPorts($itemtype, $a_inventory, $items_id) {
```

Comment:
Should not this one be replaced with PluginFusioninventoryInventoryCommon::importPorts?

Reply:
I decided not to do it, because the code of a networkequipment import is much more complex than the rest and it would take much more time.
What I can do it adding a todo not to forget

Reply:
OK for me, and OK to add a comment :)

Reply:
done